orttraining/orttraining/core/graph/training_op_defs.cc

```diff
@@ -2220,19 +2220,27 @@ Return true if all elements are true and false otherwise.
       .Output(0, "outputs_grad", "Gradient of outputs returned from pytorch.", "T", OpSchema::Variadic,
               /*is_homogeneous*/ false,
               /*min_arity*/ 1)
+      .Attr(
+        "RequiredGrad",
+        "The indices of required gradient outputs.",
```

Comment:
nit. "The input indices that required gradients."

Reply:
I think it may be better if I change it to "The indices of the required gradient outputs."